Document new only expunge deletes behavior for Force Merge API (#5837)
opensearch-project/documentation-website: _api-reference/index-apis/force-merge.md

In OpenSearch, a shard is a Lucene index, which consists of _segments_ (or segment files). Segments store the indexed data. Periodically, smaller segments are merged into larger ones and the larger segments become immutable. Merging reduces the overall number of segments on each shard and frees up disk space. 

> OpenSearch performs background segment merges that produce segments whose size is at most `index.merge.policy.max_merged_segment` (the default is 5 GB).

Reviewer: ```suggestion
OpenSearch performs background segment merges that produce segments no larger than `index.merge.policy.max_merged_segment` (the default is 5 GB).
```

> OpenSearch performs background segment merges that produce segments whose size equals to at most `index.merge.policy.max_merged_segment` (the default is 5 GB).

Reviewer 2: Maybe reword to "produce segments whose size is at most" ?

Author: This is better, thanks!

> OpenSearch performs background segment merges that produce segments whose size equals to at most `index.merge.policy.max_merged_segment` (the default is 5 GB).

Author: ```suggestion
OpenSearch performs background segment merges that produce segments whose size is at most `index.merge.policy.max_merged_segment` (the default is 5 GB).
```
